### DISCUSSION TITLE
Support for NanoPi Neo3 pls

### DISCUSSION THREAD

**Discussion by tia9os** [adafruit/Adafruit_Blinka#1067] [New Board Requests]
https://wiki.friendlyarm.com/wiki/index.php/NanoPi_NEO3

PLS add example for adding new boards <3

**Comment by jepler** [comment]
It looks like the linked computer is a Linux single-board computer, so this guide may help you get started: https://learn.adafruit.com/adding-a-single-board-computer-to-platformdetect-for-blinka

**Comment by tia9os** [comment]
Hey,

thanks for the quick reply!

What about the example?
+ It looks like there is support for the nanopi neo air but no support for the nanopi neo3.

If you give an example on how the implementation of new boards works it would be much easier to add my own.

**Comment by ladyada** [comment]
https://learn.adafruit.com/adding-a-single-board-computer-to-blinka

**Comment by ruskifilozof** [comment]
Please help with adding support for Nano Pi NEO 3. Adafruit-PlatformDetect version 3.62.0 detecting chip RK3328 correctly but "self._rock_pi_id()" returns "None", in result whole platform detection fails. I've tried to add support using link above but stuck. Any chance to get this board working? 
Thank you in advance.

**Comment by makermelissa** [comment]
@ruskifilozof yeah, the board still hasn't been added yet. if you are seeing the chip being detected correct, then that's some of the work already done for you. If you'd like to get it working, you can follow the guide that @jepler linked to above in this issue.
